src/playback/view/internal/inputresourceitem.cpp

```diff
@@ -174,4 +175,16 @@ void InputResourceItem::updateAvailableResources(const AudioResourceMetaList& av
         AudioResourceMetaList& resourcesMetaList = resourcesByVendor[meta.vendor];
         resourcesMetaList.push_back(meta);
     }
+
+    for (auto& [type, resourcesByVendor] : m_availableResourceMap) {
```

Comment:
The exact same code is presented in two places. Please don't duplicate it

Reply:
Fixed!